BuildFFMpeg.ps1

```diff
@@ -0,0 +1,37 @@
+param (
+    [ValidateSet('x64', 'x86', 'ARM')]
+    [string] $Platform = 'x64',
+
+    [string] $MSys2Dir = 'C:\msys64',
+
+    [string] $VSInstallDir = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\2017\Enterprise",
+    [string] $VCVersion = '14.16.27023',
+    [string] $VCInstallDir = "$VSInstallDir\VC\Tools\MSVC\$VCVersion",
+
+    [string] $WindowsKitsDir = "${env:ProgramFiles(x86)}\Windows Kits",
```

Comment:
This needs to be generic. The easiest thing is to use vswhere and commit a version into a tools directory on the repo